data/windows.yaml

```diff
@@ -2103,3 +2103,23 @@ urls:
 - 'https://www.symantec.com/security_response/writeup.jsp?docid=2012-020609-4221-99&tabid=2'
 - 'http://www.nirsoft.net/utils/winsock_service_providers.html'
 - 'https://msdn.microsoft.com/en-us/library/windows/desktop/ms739923(v=vs.85).aspx'
+---
+name: WindowsDNSNameServer
+doc: |
+    Windows DNS and DHCP Server Registry Keys.
+
+    Typical location for hijacking the DNS or DHCP servers of a client.
```

Comment:
> Typical location for hijacking the DNS or DHCP servers of a client.

These are the locations the DNS setting are stored in Windows. I agree that malware can change them but for the artifact itself IMHO less relevant.

Maybe change doc to:
Windows Registry Keys that contain DNS and DHCP settings.